src/Sulu/Bundle/ContactBundle/Entity/AccountRepository.php

```diff
@@ -188,7 +188,8 @@ public function findByIds($ids)
             ->addSelect('faxType')
             ->addSelect('notes')
             ->addSelect('medias')
-            ->where('account.id IN (:accountIds)');
+            ->where('account.id IN (:accountIds)')
+            ->orderBy('account.id', 'ASC');
```

Comment:
I thought about something like adding the order argument to every test where this makes problems. But I guess that is not possible everywhere... So we probably have to live with that for the moment 😕